src/lib/gundb/UserPropertiesClass.js

```diff
@@ -161,10 +174,12 @@ export default class UserProperties {
    * Reset properties to the default state
    */
   async reset() {
-    const { defaultProperties } = UserProperties
+    const { data } = this
+    const { defaultProperties, keepFields } = UserProperties
+    const dataBeenReset = assign({}, defaultProperties, pick(data || {}, keepFields))
```

Comment:
users props are removed on account removal, but invitees data is kept in blockchain
so after account removal & re-create user sees again notifications about invitees

to solve this issue i propose do not remove some of props (like invitees stats or for example flag-like props such as goodMarketClicked) on account removal (when userProperties.reset() is called)

Reply:
@sirpy  can't we reset those properties on blockchain when deleting an account?